authentication-guides/workday/README.md

```diff
@@ -107,41 +107,41 @@ Run the **`Activate Pending Security Policy Changes`** task to activate permissi
 
 Universal search for **`Register API Client for Integrations`**
 
-![https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
+![search for register api client for integrations](https://developer.moveworks.com/static/3f4721c823875fbc16322fb8cc12f78c/410f3/Untitled_10.png)
 
 Set the name to **Moveworks** and add the scopes required.
 
-![https://developer.moveworks.com/static/4c87b5210013217701570e0558eec348/c6bbc/registration.png](https://developer.moveworks.com/static/4c87b5210013217701570e0558eec348/c6bbc/registration.png)
+![registration page for settings scopes](https://developer.moveworks.com/static/4c87b5210013217701570e0558eec348/c6bbc/registration.png)
 
 Note your **`Client ID`** & **`Client Secret`**
 
-![https://developer.moveworks.com/static/5011d27344ce82ae7d3b2ddc283b1e65/c6bbc/client_secret.png](images/client_secret.png)
+![generate client secret settings page](images/client_secret.png)
 
 Navigate to **`View API Clients`**. Note the **`Token Endpoint`** and **`Workday REST API Endpoint`**
 
-![https://developer.moveworks.com/static/f83407419897649f501e24707114ec46/c6bbc/token_and_api.png](images/token_and_api.png)
+![search for "view api clients"](images/token_and_api.png)
 
 ## Step 6: Provision a Refresh Token for the ISU
 
 From the **`View API Clients`** view, click on the **`API Clients for Integrations`** tab. Pick out the API Client you just created
 
-![https://developer.moveworks.com/static/703d5272d024109eb4b30d808292e8a6/c6bbc/inspect_client.png](images/inspect_client.png)
+![view api clients with red box around "api clients for integrations"](images/inspect_client.png)
 
 From the related actions menu, select **`Manage Refresh Tokens for Integrations`**
 
-![https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png](https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png)
+![manage refresh tokens page](https://developer.moveworks.com/static/6a2406d2a729e02c9e1f214ee8a6ea35/c6bbc/Untitled_14.png)
 
 Add the ISU Account you created earlier to the API Client
 
-![https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png](https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png)
+![showing adding the isu account created earlier](https://developer.moveworks.com/static/b1bab6dffdf67d31c35640c840c0f17b/c6bbc/isu_to_client.png)
```

Comment:
Moved to https://help.moveworks.com/docs/workday-access-requirements

Reply:
brought image locally